addons/web/static/src/xml/base.xml

```diff
@@ -413,7 +413,7 @@
 </form>
 <t t-name="GraphView.buttons">
     <div class="btn-group" role="toolbar" aria-label="Main actions">
-        <button class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+        <button class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-expanded="false" data-display="static">
             Measures
         </button>
         <div class="dropdown-menu o_graph_measures_list" role="menu">
```

Comment:
LGTM. A non-fixed dropdown is much nicer. 
I'd addtionally add the class o_dropdown_menu to make it scrollable like the group by dropdown.

```suggestion
            Measures
        </button>
        <div class="o_dropdown_menu dropdown-menu o_graph_measures_list" role="menu">
```